themes/StarterTheme/config/theme.dist.yml

```diff
@@ -83,6 +83,19 @@ global_settings:
               - category
               - index
 
+  image_types:
+    # When theme will be enabled, all image types will be removed
+    # Template must declare their image type.
```

Comment:
Like we said on Gitter (I say it here for history, I mean History) this feature isn't multistore yet. We'll add this later. :)